_posts/2019-05-24-codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter.md

```diff
@@ -203,7 +203,7 @@ KnapsackPro::Hooks::Queue.after_subset_queue do |queue_id, subset_queue_id|
 end
 {% endhighlight %}
 
-This example is based on [FAQ](https://github.com/KnapsackPro/knapsack_pro-ruby#how-to-use-junit-formatter).
+This example is based on [Knapsack Pro FAQ](https://github.com/KnapsackPro/knapsack_pro-ruby#how-to-use-junit-formatter).
```

Comment:
we could link here to this page https://knapsackpro.com/faq/question/how-to-use-junit-formatter

Reply:
Good idea. Updated.